pcdsdevices/movablestand.py

```diff
@@ -16,8 +16,8 @@ class MovableStand(InOutPVStatePositioner):
 
     name: ``str``, required keyword
     """
-    in_limit = Cmp(EpicsSignalRO, ':IN_DI')
-    out_limit = Cmp(EpicsSignalRO, ':OUT_DO')
+    in_limit = Cmp(EpicsSignalRO, ':IN_DI', kind='normal')
+    out_limit = Cmp(EpicsSignalRO, ':OUT_DO', kind='normal')
```

Comment:
Are we including this every time just so that we know someone has thought about it? This is the default value after all.